jepsen/src/jepsen/checker.clj

```diff
@@ -382,34 +380,51 @@
              upper              0             ; Upper bound on counter value
              pending-reads      {}            ; Process ID -> [lower read-val]
```

Comment:
Does the comment need to be updated? It looks like we are storing [lower, upper] instead.

Reply:
on line 395.